internal/cmd/root.go

```diff
@@ -126,7 +127,7 @@ a smooth experience as a Golang developer, using the best tools in the market!`,
 	PersistentPreRunE: func(_ *cobra.Command, _ []string) error {
 		setupViperAutomaticEnv()
 		logLady = logger.GetLogger("", configOssi.LogLevel)
-		return checkForUpdates("")
```

Comment:
Basically because it was so early in the process, I didn't know the command line flags. It happens before process config, so I moved it later into the chain to occur after that.